examples/nfs/README.md

```diff
@@ -49,16 +38,21 @@ controller and import it into two replication controllers.
 Define [NFS server controller](nfs-server-rc.yaml) and
```

Comment:
For NFS server part, can we make this more descriptive and start with a verb - e.g. "Create the NFS _Service_ and _Replication Controller_"?